doc/source/math/resizing_ops.rst

```diff
@@ -0,0 +1,356 @@
+.. _resizing_ops:
+
+##################
+Resizing Operators
```

Comment:
Much of the text here is about boundary behaviors. We should probably write a separate doc on those (since this should match) and refer it from here. That can be done later.